packages/editor-ui/src/composables/useGenericHelpers.ts

```diff
@@ -0,0 +1,35 @@
+import dateformat from 'dateformat';
+import { useI18n } from './useI18n';
+
+export function useGenericHelpers() {
```

Comment:
The two functions could've probably lived as date utility functions, outside of a composable.

Reply:
That's a good point. I moved `displayTimer` function to `i18n` plugin and `convertToDisplayDate` to `dateFormatter` utils. Thanks for that!